linux-build.proj

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0"
+         DefaultTargets="Build"
+         xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <RHostName Condition=" '$(RHostName)'=='' ">Microsoft.R.Host</RHostName>
+    <RootDirectory Condition=" '$(RootDirectory)'=='' ">$(MSBuildThisFileDirectory)..\..\..\</RootDirectory>
+    <OutputDirectory Condition=" '$(OutputDirectory)'=='' ">$(RootDirectory)bin\</OutputDirectory>
+    <IntermediateOutputDirectory Condition=" '$(IntermediateOutputDirectory)'=='' ">$(RootDirectory)obj\$(RHostName)\</IntermediateOutputDirectory>
```

Comment:
In vcxproj, this currently also includes the platform:
```
    <IntDir>$(ObjDirectory)$(Configuration)\$(TargetName)\$(Platform)\Microsoft.R.Host</IntDir>
```
It's not really a big deal, but if we do build for multiple platforms, it means that intermediate object files won't be overwritten on every build if you switch back and forth.